crates/project_panel/src/project_panel.rs

```diff
@@ -872,6 +876,10 @@ impl ProjectPanel {
         }
     }
 
+    fn duplicate(&mut self, _: &Duplicate, cx: &mut ViewContext<Self>) {
+        todo!()
+    }
```

Comment:
Currently Im here trying to use the `duplicate` method introduced here: https://github.com/zed-industries/zed/pull/7123/files#diff-e13638ad3885e643f14ba8cde1f20fb4955f2cf025ee6b4e7bdb33a0c3f00e50R901-R906